src/RoveComm/RoveCommUDP.h

```diff
@@ -0,0 +1,100 @@
+/******************************************************************************
+ * @brief The RoveCommUDP class is used to send and receive data over a UDP
+ *        connection.
+ *
+ * @file RoveCommUDP.h
+ * @author Eli Byrd (edbgkk@mst.edu)
+ * @date 2024-02-06
+ *
+ * @copyright Copyright Mars Rover Design Team 2024 - All Rights Reserved
+ ******************************************************************************/
+
+#ifndef ROVECOMM_UDP_H
+#define ROVECOMM_UDP_H
+
+#include <arpa/inet.h>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.